### ISSUE TITLE
Lint/ReturnInVoidContext raises an error

### ISSUE THREAD

**Issue by frodsan** [bbatsov/rubocop#4736]
## Expected behavior

Provide warning.

## Actual behavior

Raises error message:

```
An error occurred while Lint/ReturnInVoidContext cop was inspecting /Users/me/projects/project/config/initializers/sentry.rb:5:0.
An error occurred while Lint/ReturnInVoidContext cop was inspecting /Users/me/projects/project/config/initializers/rack_attack.rb:5:0.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.50.0 (using Parser 2.4.0.0, running on ruby 2.4.2 x86_64-darwin16)
```

## Steps to reproduce the problem

I think it fails with files like this:

```ruby
# frozen_string_literal: true

# Be sure to restart your server when you modify this file.

return if Rails.env.test? # I think it fails because of this.

# Set Rack::Attack middleware.
Rails.application.config.middleware.use(Rack::Attack)
```

## RuboCop version

```
$ rubocop -V
0.50.0 (using Parser 2.4.0.0, running on ruby 2.4.2 x86_64-darwin16)
```

**Comment by atheiman** [comment]
@bbatsov thanks for fixing this, do you know when 0.50.1 will be released?

**Comment by mikegee** [comment]
@atheiman [0.51.0](https://rubygems.org/gems/rubocop/versions/0.51.0) was released a couple days ago.
